### PR TITLE
Add Sam Saccone story 🆒

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: data/data.json [hunk 1]
```diff
@@ -548,6 +548,15 @@
...
       "story": "Rejected by Valve Software for a Software Engineer position because i was young. (when i was 19)",
       "double": false
+    },
+    {
+      "fullName": "Sam Saccone",
+      "handle": "samccone",
+      "website": "http://samsaccone.com",
+      "bio": "🐐",
+      "avatar": "samccone.jpeg",
+      "story": "I could not even get an unpaid internship at a newspaper because I was going to add too much overhead. 🆒",
```
Comment (thejameskyle) [inline data/data.json]: :arrow_upper_right: :ok_woman:

Comment (samccone) [inline data/data.json]: :tophat:

Comment (thejameskyle) [inline data/data.json]: :hourglass_flowing_sand: :hourglass: :dancer:

Comment (samccone) [inline data/data.json]: :hourglass: :two: :dancer: :ok: :round_pushpin: :question:

Comment (samccone) [inline data/data.json]: @sebmck :heavy_plus_sign: :rooster: :nose: :floppy_disk: :pig:

Comment (thejameskyle) [inline data/data.json]: @sebmck :earth_asia: :arrow_lower_left: = :koala: :crocodile: :snake: :skull: & :fried_shrimp: :on: :barber:

**Comment by jkup** [comment]
lol I don't think I speak emoji well enough to understand what's happening but thanks for adding your story!!

File: data/data.json (hunk 1 above)
Comment (sebmck) [inline data/data.json]: :open_mouth: :angry: :facepunch: :dancer: :baby:

Comment (samccone) [inline data/data.json]: @sebmck :mag: :airplane: :candy: :candy: :lollipop: :mask: :skull: :police_car: :computer: :fire: :toilet:

**Pull request merged**